docs/docs/quick-start.md

```diff
@@ -4,47 +4,50 @@ description: Redwood quick start
 
 # Quick Start
 
-> **Prerequisites**
->
-> - Redwood requires [Node.js](https://nodejs.org/en/) (>=16.19 <=18.x) and [Yarn](https://yarnpkg.com/) (>=1.15)
-> - Are you on Windows? For best results, follow our [Windows development setup](how-to/windows-development-setup.md) guide
```

Comment:
These asides (with the `>`) render as a light red right now which isn't a great color for a quick start when you want to convey how well things are going to go go:

<img width="958" alt="image" src="https://user-images.githubusercontent.com/32992335/236957278-427e6b80-936e-48df-ba27-fd7a4ae125bd.png">

Reply:
Now that you pointed it out, I can't stop thinking "gross" 🤢

Good call. Doesn't need to be a part of this PR, to be clear.

Reply:
@thedavidprice this was a Rachel Nabors tip!